speechbrain/decoders/seq2seq.py

```diff
@@ -1394,9 +1397,9 @@ def permute_lm_mem(self, memory, index):
     def forward_step(self, inp_tokens, memory, enc_states, enc_lens):
         """Performs a step in the implemented beamsearcher."""
         memory = _update_mem(inp_tokens, memory)
-        dec_out = self.model.forward_decoder(enc_states, memory)
+        dec_out, attn, = self.module.forward_decoder(enc_states, memory)
         log_probs = self.softmax(dec_out[:, -1])
-        return log_probs, memory, None
```

Comment:
We should make sure that this won't break any model

Reply:
I just followed what we did for the S2STransformerBeamSearch. The beam search algorithm expects to get the attn weights if provided. In our case, we have attn weights, so it makes sense to return them. One note: I think the return of attn from the decoder should be mandatory (and not optional). 

I did some tests with and without attn in the beam search and did not see any changes. (might be related to how I initialized the beam search. I will investigate later tomorrow).